lib/htmlValidator.js

```diff
@@ -46,31 +49,43 @@ module.exports = function(app, callback) {
           'java', ['-Xss1024k', '-cp', vnu, 'nu.validator.servlet.Main', params.htmlValidator.port ? params.htmlValidator.port : '8888']
         );
 
+        console.log(('Validator intializing...').yellow);
+
         validatorProcess.stdout.on('data', (data) => {
           if (`${data}`.includes('Initialization complete')) {
+            clearTimeout(validatorTimeout);
+            console.log(('Validator initialization completed successfully.').green);
```

Comment:
@Autre31415 I added a line to tell the user when initialization was successful, should we keep this or just get rid of it?